libcloud/compute/drivers/openstack.py

```diff
@@ -2020,8 +2021,8 @@ def _to_node(self, api_node):
                 key_name=api_node.get('key_name', None),
                 disk_config=api_node.get('OS-DCF:diskConfig', None),
                 config_drive=config_drive,
-                availability_zone=api_node.get('OS-EXT-AZ:availability_zone',
-                                               None),
```

Comment:
Next time please this as it was - we prefer to go with "explicit over implicit" and always include default value when using dict.get.